scripts/fedbiomed_vpn

```diff
@@ -221,7 +232,7 @@ containers_clean() {
 
     for i in base basenode
     do
-        docker-compose rm -sf $i >/dev/null
+        docker compose rm -sf $i >/dev/null
```

Comment:
We can also add a warning here regarding old `docker-compose`.

Reply:
Done for all instances of `docker compose` in (hopefully) all scripts using it.